pkg/provisioners/template.go

```diff
@@ -412,6 +422,7 @@ func renderTemplateString(str string, entry *registry.Entry, data interface{}) (
 		"snippetArray":        templateFunctionSnippetArray(entry),
 		"list":                templateFunctionList,
 		"generatePassword":    templateFunctionGeneratePassword,
+		"generatePetName":     templateFunctionGeneratePetName,
```

Comment:
To paraphrase Paris Hilton: Loves it!  Thanks for the contribution.

Would you be so kind as to update the end user documentation to, communicate this exists to new users?

Reply:
Updated the documentation, working out how to build the documentation was another problem entirely